clients/admin-ui/src/features/config-wizard/organization.slice.ts

```diff
@@ -0,0 +1,143 @@
+import { createSlice, PayloadAction } from "@reduxjs/toolkit";
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+import { HYDRATE } from "next-redux-wrapper";
+
+import type { AppState } from "../../app/store";
+import {
+  Organization,
+  OrganizationParams,
+  OrganizationResponse,
+} from "./types";
+
+export interface State {
+  page: number;
+  size: number;
+  token: string | null;
```

Comment:
I'm not sure if it's needed in this PR, but it seems like we should store organization info in the State?